metric-anomaly-data-model/build.gradle.kts

```diff
@@ -11,7 +11,7 @@ dependencies {
     api("javax.annotation:javax.annotation-api:1.3.2")
     api("org.apache.avro:avro:1.10.2")
     implementation("com.typesafe:config:1.4.1")
-    implementation("org.apache.kafka:kafka-clients:2.6.0")
+    implementation("org.apache.kafka:kafka-clients:2.8.1")
```

Comment:
We have compatibility issue related to Kafka version - https://docs.confluent.io/platform/current/installation/versions-interoperability.html

And, I guess, we had some issue with streaming so, we have been using `6.0.x` version of the confluent library.

Reply:
Please don't upgrade Kafka libs. We have a functional issue (no BOLA events) which is still under investigation.